netbox/utilities/templates/search/searchbar.html

```diff
@@ -5,7 +5,11 @@
     aria-label="Search"
     placeholder="Search"
     class="form-control"
-    value="{{ request.GET.q }}"
+    value=""
```

Comment:
Why not leave this as-is?

Reply:
By setting the value with a javascript, the cursor moves to the end of the input field.. if I just set the value, the cursor is at the start of the input field...

Reply:
But it doesn't even render the value until the user clicks the field, which doesn't resolve #8813. Just leaving it as `value="{{ request.GET.q }}"` ensures the value is populated, and the cursor defaults to the end of the string on focus.

Reply:
The autofocus part takes care of setting the focus. 
If I leave out the onfocus part. then the cursor is at the start of the value.
You need to merge that PR #8908 to make this work. The autofocus is currently overruled by the piece of javascript in the netbox.ts

Reply:
Ok. For now, can we change this to `value="{{ request.GET.q }}"` just to close out #8813? I suspect #8908 might need some additional tweaking but I'm waiting for Matt to weigh in. We can add the autofocus bit under that PR to ensure it's correlated with the appropriate FR.

Reply:
I removed the autocomplete and onfocus from this PR.. Moving that to the #8908